hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-common/src/main/java/org/apache/hadoop/yarn/server/federation/policies/amrmproxy/LocalityMulticastAMRMProxyPolicy.java

```diff
@@ -195,9 +198,14 @@ public void notifyOfResponse(SubClusterId subClusterId,
       AllocateResponse response) throws YarnException {
     if (response.getAvailableResources() != null) {
       headroom.put(subClusterId, response.getAvailableResources());
-      LOG.info("Subcluster {} updated with {} memory headroom", subClusterId,
-          response.getAvailableResources().getMemorySize());
     }
+    if (response.getEnhancedHeadroom() != null) {
+      this.enhancedHeadroom.put(subClusterId, response.getEnhancedHeadroom());
```

Comment:
Shouldn't we do something with this?

Reply:
This will be used to split resource request based on cluster load. This is prereq for this JIRA: https://issues.apache.org/jira/browse/YARN-11037
Will be using enhancedHeadroom to calaculate load on a sub cluster.

Reply:
I think it should be fine to break changes into two JIRAs for ease of validations. What are your thoughts ? Should I merge them into one PR only ?

Reply:
It does make sense. Let's just have the links.